lang/en/lang.php

```diff
@@ -58,3 +58,8 @@
 $lang['preparingdownload']      = "We are preparing your download, please wait...";
 $lang['titleerrordownload']     = "Error";
 $lang['faileddownload']         = "There was a problem generating your download.";
+
+$lang['select_namespace']       = "Add Namespace to Selection";
```

Comment:
- This button opens a window. What about "Add a Namespace…" (or "Add Namespace to Selection…")
- And I'm wondering if a button is the most clear/nice gui element. 
Maybe a link like "Add this page to your book" with +symbol `images/add.png`  is nicer?
(note: that clear button at the bottom is something at my future todo list for this bookmanager gui ;-)